catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -193,7 +198,9 @@ public QueryResponse federate(List<Source> sources, QueryRequest queryRequest) {
     }
 
     QueryResponse queryCache(QueryRequest queryRequest) {
-        QueryRequest updatedRequest = getRequestWithTagsFilter(queryRequest);
+        QueryRequest updatedRequest = validationQueryFactory.getQueryRequestWithValidationFilter(
```

Comment:
What's wrong with just pulling both instances out into a single `private QueryRequest doCustomLogic()` method? I think that's what @pklinef is suggesting.